bootstrap-joining-demo/local/src/main/resources/second.conf

```diff
@@ -0,0 +1,39 @@
+akka {
+
+  actor {
+    provider = "cluster"
+  }
+
+  remote {
+    artery {
+      enabled = on
+      transport = tcp
+      canonical.hostname = "akka2"
+      canonical.port = 25552
+    }
+  }
+
+  discovery {
+    method = akka-dns
+  }
+
+  management {
+    http {
+      port = 19999
```

Comment:
isn't this already taken by `first`?

Reply:
They are listening on different interfaces akka1/2/3 which are 127.0.1/2/3

Reply:
Please use the 8558 port? we moved everything to it in akka-management